src/main/java/com/easypost/model/Fee.java

```diff
@@ -1,36 +1,87 @@
+/**
+ * Fee.java
+ * This file is a part of EasyPost API SDK.
+ * (c) 2022 EasyPost
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
 package com.easypost.model;
 
-public class Fee{
-    String type;
-    float amount;
-    Boolean charged;
-    Boolean refunded;
+public final class Fee {
+    private String type;
+    private float amount;
+    private Boolean charged;
+    private Boolean refunded;
 
+    /**
+     * Get Fee type.
+     *
+     * @return type
+     */
     public String getType() {
         return type;
     }
-    public void setType(String type) {
+
+    /**
+     * Set Fee type.
+     *
+     * @param type type
+     */
+    public void setType(final String type) {
         this.type = type;
     }
 
+    /**
+     * Get Fee amount.
+     *
+     * @return amount
+     */
     public float getAmount() {
         return amount;
     }
-    public void setAmount(float amount) {
+
+    /**
+     * Set Fee amount.
+     *
+     * @param amount amount
+     */
+    public void setAmount(final float amount) {
         this.amount = amount;
     }
 
+    /**
+     * Get if the fee is charged.
```

Comment:
These are similarly confusing as the `refund` verbiage was below. Can we tweak these too so the description doesn't sound conditional for the getter and setter?

Reply:
All boolean getters/setters docstrings like this have been clarified in aed8e7f